src/client/inputhandler.cpp

```diff
@@ -142,7 +142,8 @@ bool MyEventReceiver::OnEvent(const SEvent &event)
 		if (event.JoystickEvent != joystick_we_listen_for)
 			return false;
 		*/
-		return joystick->handleEvent(event.JoystickEvent);
+		// joystick may be nullptr if game is launched with '--random-input' parameter
```

Comment:
Please also remove the comment above, the joystick handler does that internally so it doesn't apply here.

Reply:
I removed my comment. Should I also remove the block comment with the `TODO add a check like:`?

Reply:
Uh yes, that's the comment I meant in the first place.

Reply:
got my comment back and removed the block-comment